docs/releases.rst

```diff
@@ -3,6 +3,56 @@
 Release notes
 =============
 
+Version 3.0.3
+-------------
+
+**New features**
+
+- Compatibility with PostgreSQL 16 beta1 (Alexander Kukushkin)
+
+  Extended GUC's validator rules.
+
+- Make PostgreSQL GUC's validator extensible (Israel Barth Rubio)
+
+  Validator rules are loaded from YAML files located in ``patroni/postgresql/available_parameters/`` directory. Files are ordered in alphabetical order and applied one after another. It makes possible to have custom validators for non-standard Postgres distributions.
+
+- Added ``restapi.request_queue_size`` option (Andrey Zhidenkov)
+
+  Sets request queue size for TCP socket used by Patroni REST API. Once the queue is full, further requests get a "Connection denied" error. The default value is 5.
+
+- Call ``initdb`` directly when initializing a new cluster (Matt Baker)
+
+  Previously it was called via ``pg_ctl``, what required a special quoting of parameters passed to ``initdb``.
+
+- Added before stop hook (Le Duane)
+
+  The hook could be configured via ``postgresql.before_stop`` and is executed right before ``pg_ctl stop``. The exit code doesn't impact shutdown process.
+
+- Added support for custom Postgres binary names (Israel, Polina Bungina)
```

Comment:
We should put full names

Reply:
We actually only put full names at the first mention. Is that a problem?

Reply:
The problem I see is consistency overall. I read "Polina Bungina" in one place and "Polina" later in the Release Notes and that would at least raise an eye-brow.
Is there a reason not to have the full name on every feature/bug fix? The Postgres release notes follow the same convention, which I've always found pretty nice overall (everything in the Postgres docs is very well written and organized).

Reply:
I see and ultimately agree. I will change it in a bit, hope @CyberDem0n doesn't mind.